povm_toolbox/post_processor/povm_post_processor.py

```diff
@@ -15,28 +15,47 @@
 import numpy as np
 from qiskit.quantum_info import SparsePauliOp
 
+from povm_toolbox.quantum_info.base_dual import BaseDUAL
+from povm_toolbox.quantum_info.multi_qubit_dual import MultiQubitDUAL
+from povm_toolbox.quantum_info.multi_qubit_povm import MultiQubitPOVM
+from povm_toolbox.quantum_info.product_dual import ProductDUAL
+from povm_toolbox.quantum_info.product_povm import ProductPOVM
 from povm_toolbox.sampler import POVMPubResult
 
 
-class POVMPostprocessor:
+class POVMPostProcessor:
     """A common POVM result post-processor."""
 
     def __init__(
         self,
         povm_sample: POVMPubResult,
-        alphas: np.ndarray | None = None,
+        DUAL_CLASS: type[BaseDUAL] | None = None,
```

Comment:
We should not use all-caps variable names since the convention is that all-caps named objects are constants.

Reply:
Yes sorry I'll change it